### PR TITLE
Added methods Min and Max to Decimal.

### PR DESCRIPTION
**Pull request by gjvnq**

### CODE REVIEW

File: decimal.go [hunk 1]
```diff
@@ -447,6 +447,26 @@ func (d Decimal) MarshalText() (text []byte, err error) {
 	return []byte(d.String()), nil
 }
 
+func (d Decimal) Min(others ...Decimal) Decimal {
```
Comment (vadimg) [inline decimal.go]: I would prefer these to be functions, not methods. It looks much better to do `decimal.Min(a, b, c)` than to do `a.Min(b, c)`

File: decimal_test.go [hunk 2]
```diff
@@ -449,6 +449,26 @@ func TestDecimal_Cmp2(t *testing.T) {
...
+		t.Errorf("Error")
+	}
+}
+
+func TestDecimal_Max(t *testing.T) {
+	a := NewFromFloat(3.14)
+	b := NewFromFloat(1.6)
+	c := NewFromFloat(-2)
+
+	if a.Max(b, c) != a {
```
Comment (vadimg) [inline decimal_test.go]: You shouldn't use == or != to test equality of Decimals, you should use .Equals

File: decimal.go (hunk 1 above)
Comment (gjvnq) [inline decimal.go]: I've fixed that on commit 65c6084.

**Comment by vadimg** [comment]
This code is not up to the standards of this library. For example, here are some issues:
- What happens when it's called with no arguments?
- The tests are not thorough enough
- There's no documentation
- Why do you check for <= and >= instead of < and >?

I decided it would be easier and faster for me to just implement this myself.

**Pull request closed without merging**